Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Scene.h

```diff
@@ -176,7 +176,8 @@ namespace AZ
         protected:
             // SceneFinder overrides...
             void OnSceneNotifictaionHandlerConnected(SceneNotification* handler);
-                        
+            void PipelineStateLookupNeedsRebuild();
```

Comment:
These should be followed by override keyword

Reply:
done.